ui-tests/src/test/resources/features/integrations/api-provider.feature

```diff
@@ -176,17 +177,27 @@ Feature: API Provider Integration
       | id        | body.id        |
       | completed | body.completed |
       | task      | body.task      |
-    And sleep for jenkins delay or "2" seconds
     And check "Done" button is "visible"
     And click on the "Done" button
 
+    And edit integration step on position 5
+    And map Api Provider errors
+      | Server Error         | 404 |
```

Comment:
I'd suggest setting a different error code here to see that it does not return 404 for everything now :D

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418 :grin: